SConstruct

```diff
@@ -1976,10 +1973,12 @@ if env['f90_interface'] == 'y':
 VariantDir('build/src', 'src', duplicate=0)
 SConscript('build/src/SConscript')
 
-if env['python_package'] == 'full':
-    SConscript('interfaces/cython/SConscript')
-elif env['python_package'] == 'minimal':
-    SConscript('interfaces/python_minimal/SConscript')
+if env["python_package"] == "full":
+    VariantDir("build/python", "interfaces/cython", duplicate=True)
+    SConscript("build/python/SConscript")
```

Comment:
That's unfortunate... For me, it's so much easier to reason about actual files than abstract file nodes. I think it also streamlines the installation method (pip), which doesn't have to account for the built library being in a different location, and which also doesn't know anything about file nodes.

When possible, SCons hard links files rather than copying them. However, most editors (to my understanding) use a copy-then-move strategy to save files, which creates a new inode and breaks the hardlink, so that doesn't actually help us.

What I've done is use `pytest-watch`, which allows automatically running a command before running the test suite whenever a file is saved. This doesn't support the interactive feature development case so much, but it's better than nothing.

Reply:
@speth Do you have any further thoughts here?